build.py

```diff
@@ -320,13 +338,13 @@ def releaseId() :
 	# preferred python from the environment. SCons itself
 	# unfortunately hardcodes `/usr/bin/python`, which might not
 	# have the modules we need to build the docs.
-	buildCommand = "python `which scons` package PACKAGE_FILE={uploadFile} ENV_VARS_TO_IMPORT=PATH DELIGHT_ROOT={delight} ARNOLD_ROOT={arnoldRoot} OPTIONS='' -j {cpus}".format(
+	buildCommand = "python `which scons` package PACKAGE_FILE={uploadFile} ENV_VARS_TO_IMPORT=PATH DELIGHT_ROOT={delight} ARNOLD_ROOT={arnoldRoot} RENDERMAN_ROOT={renderManRoot} OPTIONS='' -j {cpus}".format(
 		cpus=multiprocessing.cpu_count(), **formatVariables
 	)
 
 else :
 
-	buildCommand = "env RMAN_ROOT={delight} ARNOLD_ROOT={arnoldRoot} BUILD_DIR={cwd}/gafferDependenciesBuild ./build/buildAll.sh ".format(
+	buildCommand = "env RMAN_ROOT={delight} ARNOLD_ROOT={arnoldRoot} RENDERMAN_ROOT={renderManRoot} BUILD_DIR={cwd}/gafferDependenciesBuild ./build/buildAll.sh ".format(
```

Comment:
This is a bit of an unfortunate collision isn't it :( I haven't fully unpicked how we're defining the difference between `RMAN_ROOT` and `RENDERMAN_ROOT` upstream here, and how we're coexisting 3Delight and prman in a build. Is there any way we can refactor those vars to make it less confoozing?

Reply:
Now that you mention it, yes :disappointed:. I actually shouldn't have changed the build command for the `project == "dependencies"` case, because RENDERMAN_ROOT isn't needed there, it's just needed for Gaffer. I've pushed a change to that effect, which helps somewhat.

The variables are as follows :

- DELIGHT_ROOT : Needed to build GafferDelight. Not used by dependencies builds.
- RENDERMAN_ROOT : Needed to build GafferRenderMan. Not used by dependencies builds.
- ARNOLD_ROOT : Needed to build GafferArnold and IECoreArnold. Used by dependencies builds.
- RMAN_ROOT : Needed to build the renderman display driver in the dependencies builds. Can in theory point to either 3delight or RenderMan, although I've only tested with 3delight. The resulting binary can be used with either renderer fine. Only used by dependencies builds.